core/layout_api.php

```diff
@@ -285,6 +288,16 @@ function layout_head_css() {
 	echo "\n";
 }
 
+/**
+ * Print user font preference
+ * @return void
+ */
+function layout_user_font_preference() {
+	echo '<style>', "\n";
+    echo  '* { font-family: "' , config_get( 'font_family' ) , '"; } ', "\n";
```

Comment:
Assign to variable and use it for the two calls
```
$t_font_family = config_get( 'font_family' );
```

Do we want/need a project specific setting?
If not, call config_get with ALL_PROJECTS.